config/crd/bases/exp.infrastructure.cluster.x-k8s.io_awsmanagedclusters.yaml

```diff
@@ -0,0 +1,86 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.9
+  creationTimestamp: null
+  name: awsmanagedclusters.exp.infrastructure.cluster.x-k8s.io
+spec:
+  group: exp.infrastructure.cluster.x-k8s.io
```

Comment:
I'd suggest to use a different group, and avoid using `exp` in the package name. In a future release, we might want to promote these API types outside the experimental package and that would be much harder to do if the group differs.

One suggestion I have is to use either start from `v1alpha1` and use `eks.aws.infrastructure.cluster.x-k8s.io`, or `v1alpha3` and use `aws.infrastructure.cluster.x-k8s.io`.

Reply:
i'm more a fan of `v1alpha3` and `aws.infrastructure.cluster.x-k8s.io` personally. if we can agree to that I'll also follow a similar pattern in our upcoming AWSMachinePool PR.

Reply:
awsmachine’s group is `infrastructure.cluster.x-k8s.io` but
awsmachinepool will be `aws.infrastructure.cluster.x-k8s.io`?

Reply:
hmm great point... AWSCluster and AWSManagedCluster should probably be in the same API group 🤔

Reply:
I am fine keeping it within the same API group

Reply:
Yes, very interesting and a good point.

Shall i will change the group to `infrastructure.cluster.x-k8s.io` to match the existing resource kinds. Now its been pointed out that makes more sense to me as we will want to promote these out of `exp` into the main `api` at some point in the future.

Reply:
Thank you!

Reply:
I've changed this to `infrastructure.cluster.x-k8s.io`